hadoop-ozone/dist/src/shell/ozone/ozone

```diff
@@ -31,6 +31,8 @@ function hadoop_usage
   hadoop_add_option "--hosts filename" "list of hosts to use in worker mode"
   hadoop_add_option "--loglevel level" "set the log4j level for this command"
   hadoop_add_option "--workers" "turn on worker mode"
+  hadoop_add_option "--jvmargs" "jvm arguments, append JVM options to any existing options defined in the HADOOP_OPTS environment variable. Any defined in HADOOP_CLIENT_OPTS will be append after these jvmargs"
```

Comment:
To keep this consistent with the other options, I think this new option should be:

```
hadoop_add_option "--jvmargs jvm arguments", "append JVM options to any existing options defined in the HADOOP_OPTS environment variable. Any defined in HADOOP_CLIENT_OPTS will be append after these jvmargs"
```

Please also remove the extra blank line after this new line of code.